index.html

```diff
@@ -0,0 +1,173 @@
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- Header Section -->
+  <div>
+    <header class="nav"> 
+      <div class="travel">
+          <nav  class="span">Travel</nav>
+                <div class="harmburger-menu">
+                  <img src="asset/mobile-menu.svg" class="frame">
+        </div>
+          </div>
+            </div>
+      </header>
+      <div class="journey"> 
+          <h1><span class="span">Travel</span> around<br>the world</h1>
+      </div>
+      <div class="image-container">
+            <img src="asset/Blurry (1).svg" class="blurry">
+            <img src="asset/Ellipse (2).svg" class="ellipse">
+            <img src="asset/Image (1).svg" class="image-A">
+        </div>
+      <section class="destination">
+            <p>Find your best destination</p>
+            <h1>We have more than 1000+ destinations you can choose.</h1>
+      </section>
+      <section class="container">
+          <div class="lake">
+              <img class="rotate" src="asset/mirror.svg" alt="">
+                <h2 class="name">Mirror lake</h2>
+          <div class="location">
+                <img src="asset/Location.svg" alt="">
+                <h3 class="country">Jawa Tengah, Indonesia</h3>
+          <div class="lake">
+                  <img class="rotate"  src="asset/Hot Air.svg">
+                  <h4 class="name">Hot Air Ballon</h4>
+            <div class="location">
+                    <img src="asset/Location.svg" alt="">
+                    <h3 class="country">Cappadocia, Turkey</h3>
+              <div class="lake">
+                      <img class="rotate" src="asset/oeschinen lake.svg.svg" alt="">
+                      <h2 class="name">Oeschinen Lake</h2>
+              <div class="location">
+                        <img src="asset/Location.svg" alt="">
+                        <h3 class="country">Pasuruan, Indonesia</h3>
+              <div class="lake">
+                          <img class="rotate" src="asset/rhine falls.svg" alt="">
+                          <h2 class="name">Rhine falls</h2>
+              <div class="location">
+                            <img src="asset/Location.svg" alt="">
+                            <h3 class="country">Jawa Tengah, Indonesia</h3>
+              <div class="lake">
+                              <img class="rotate" src="asset/sunter lake.svg" alt="">
+                              <h4 class="name">Sunter Lake</h4>
+              <div class="location">
+                                <img src="asset/Location.svg" alt="">
+                                <h3 class="country">Jawa Tengah, Indonesia</h3>
+              <div class="lake">
+                                  <img class="rotate" src="asset/negeri.svg" alt="">
+                                  <h5 class="name">Negeri di atas Awan</h5>
+              <div class="location">
```

Comment:
Why are your closing div tags arranged like this? This method isn't applicable for best practice, please make sure to rectify it